README.md

```diff
@@ -25,7 +25,7 @@ Before you start using this Library, you **need** to know how PHP works, you nee
 
 ### Requirements
 
-- [PHP 8.0](https://php.net) or higher (latest version recommended)
+- [PHP 8.2](https://php.net) or higher (latest version recommended)
```

Comment:
Our composer.json still requires ^8.0, no? Why is the min version being updated?

Reply:
```suggestion
- [PHP 8.0](https://php.net) or higher (latest version recommended)
```